firmware/esp32_ttgo/DisasterServer.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DISASTERSERVER_H
```

Comment:
Why is there an empty DisasterServer.cpp? Is it just a placeholder? It doesn't appear to be required to build the firmware.

Reply:
Yeah I think that can be removed.